src/OSVRTrackedDevice.cpp

```diff
@@ -161,23 +162,127 @@ void OSVRTrackedDevice::DebugRequest(const char* request, char* response_buffer,
     // make use of (from vrtypes.h) static const uint32_t k_unMaxDriverDebugResponseSize = 32768;
 }
 
+#ifdef _WINDOWS
+// Copyright Razer LLC 2016
```

Comment:
Typical convention if needed is to add a line to the copyright header that should already be in this file, and or expand it to .... and OSVR Community